src/components/Dialog/src/DialogLayer.vue

```diff
@@ -165,6 +165,12 @@ export default {
 	}
 }
 
+@media screen and (--for-tablet-landscape-down) {
```

Comment:
We should flip this so it's mobile first. 100% should be the default and the tablet portrait up should be a set width.

Reply:
See: https://github.com/square/maker/blob/master/src/components/Modal/src/Modal.vue#L118-L124

Reply:
Hmm, actually the ModalLayer example: https://github.com/square/maker/blob/master/src/components/Modal/src/ModalLayer.vue#L244-L259

Reply:
We should actually look into internalizing a standard Layer component so we can offset this support into a single component rather than duplicating code and dealing with these issues this PR is looking to resolve. (But that will have to happen later)